ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Flex, FlexItem, PageSection } from '@patternfly/react-core';
+
+import useURLSearch from 'hooks/useURLSearch';
+import { fetchClustersAsArray } from 'services/ClustersService';
+import { Cluster } from 'types/cluster.proto';
+import ClusterSelect from './ClusterSelect';
+import NamespaceSelect from './NamespaceSelect';
+
+export type Namespace = {
+    metadata: {
+        id: string;
+        name: string;
+    };
+};
+
+function NetworkGraphPage() {
```

Comment:
Overall, the URL-driven Network Graph looks good. I can select a cluster and namespace and see the URL change